fedimint-eventlog/src/lib.rs

```diff
@@ -42,6 +42,13 @@ pub const DB_KEY_PREFIX_UNORDERED_EVENT_LOG: u8 = 0x3a;
 pub const DB_KEY_PREFIX_EVENT_LOG: u8 = 0x39;
 pub const DB_KEY_PREFIX_EVENT_LOG_TRIMABLE: u8 = 0x41;
 
+/// Minimum age in ID count for trimable events to be deleted
+const TRIMABLE_EVENTLOG_MIN_ID_AGE: u64 = 10_000;
+/// Minimum age in microseconds for trimable events to be deleted (14 days)
+const TRIMABLE_EVENTLOG_MIN_TS_AGE: u64 = 14 * 24 * 60 * 60 * 1_000_000;
```

Comment:
IMO the consumer should be responsible for trimming after they are done reading. 

or event log should maintain the position of each consumer in db, and we trim to this position which all consumers have read.

Reply:
@maan2003 Maybe, but then it's stateful, and then a single consumer that stopped consuming can lead to log no longer being trimmed, and then yhou need debugging it, some facilities to help with it, etc.

While in practice a sufficiently large retention setting guarantee that none of it is necessary.